video/out/hwdec/hwdec_cuda_gl.c

```diff
@@ -106,6 +106,18 @@ static void cuda_ext_gl_uninit(const struct ra_hwdec_mapper *mapper, int n)
 #undef CHECK_CU
 #define CHECK_CU(x) check_cu(hw, (x), #x)
 
+bool cuda_gl_check(const struct ra_hwdec *hw) {
+    if (ra_is_gl(hw->ra_ctx->ra)) {
+        GL *gl = ra_gl_get(hw->ra_ctx->ra);
+        if (gl->version < 210 && gl->es < 300) {
+            MP_VERBOSE(hw, "need OpenGL >= 2.1 or OpenGL-ES >= 3.0\n");
+            return false;
+        }
+    }
+    // This is not an OpenGL RA.
+    return false;
```

Comment:
This can be removed from `cuda_gl_init`. (GH doesn't let you comment on unchanged code so I can't highlight the lines to be removed)

Reply:
Nice catch! Fixed with https://github.com/jrelvas-ipc/mpv/commit/3b77fccfc804e3e565864f8da289d2034dc19245